nemo/collections/asr/parts/mixins/mixins.py

```diff
@@ -846,6 +852,21 @@ def _setup_streaming_transcribe_dataloader(
 
 
 class DiarizationMixin(ABC):
+
+    @staticmethod
+    def path2audio_files_to_manifest(paths2audio_files, manifest_filepath):
+        """
+        Takes paths to audio files and manifest filepath and creates manifest file with the audios
+        Args:
+            paths2audio_files: paths to audio fragment to be verified
+            manifest_filepath: path to manifest file to bre created
+        """
+        with open(manifest_filepath, 'w', encoding='utf-8') as fp:
+            for audio_file in paths2audio_files:
+                audio_file = audio_file.strip()
+                entry = {'audio_filepath': audio_file, 'offset': 0.0, 'duration': 5000, 'text': '-', 'label': 'infer'}
```

Comment:
```suggestion
                entry = {'audio_filepath': audio_file, 'offset': 0.0, 'duration': None, 'text': '-', 'label': 'infer'}
```

Reply:
leads to an error in /home/msekoyan/projects/NeMo/nemo/collections/common/parts/preprocessing/collections.py SpeechLabel __init__ when going over the data and comparing duration with min_duration and max_duration and calculating total duration
should I handle the case of "None" duration in that function?

Reply:
do you pass `min_duration` and `max_duration` values, by default these values are None? In that case how would 5000 seconds would help? It would always be over min_duration right?

Reply:
oh, sorry, for confusing
it happens when it tries calculate total_duration
"""total_duration += duration
TypeError: unsupported operand type(s) for +=: 'float' and 'NoneType'"""

Reply:
can pass the duration as "0", is it okay?

Reply:
Then skip computing of `total_duration` when `duration` is `None`. Accordingly handle the logging of the `total_duration`

Reply:
done